src/main/java/eu/tsystems/mms/tic/testerra/plugins/selenoid/webdriver/SelenoidCapabilityProvider.java

```diff
@@ -87,7 +87,10 @@ private Capabilities provide(DesktopWebDriverRequest request) {
         final String runConfigName = contextController.getExecutionContext().getRunConfig().RUNCFG;
 
         // Try to find out the current testmethod to add the name to the Selenoid caps
-        String methodName = contextController.getCurrentMethodContext().get().getName();
+
+        String methodName = contextController.getCurrentMethodContext().isPresent()
```

Comment:
`final var methodName = contextController.getCurrentMethodContext().map(AbstractContext::getName).orElse("na.")`?

Reply:
I asume you like streams? Of course, this is fancier. 😄

Reply:
No streams here. This is [Optional#map](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html#map(java.util.function.Function)).

Reply:
Yes... you know what I mean. ;-)